docs/architecture/specs/nft/README.md

```diff
@@ -0,0 +1,291 @@
+* The owner or a provider can trigger the TransferNFTCondition condition to approve the purchase and make the transfer of the NFT
+* Anyone can call the new EscrowPayment condition. It will be in charge of:
+  - Distribute the rewards to the publisher or seller
+  - Distribute the royalties to the original creator (if is not the same than the seller)
+
+Using the existing `NFTHolder` condition it will be possible to the new NFT owner to get access to Nevermined services
+
+![Publishing Flow](images/nft_publishing_flow.png)
+
+
+### Templates
+
+The Nevermined Service Execution Agreements provide standard scenarios for providing access and trigger remote computation.
+To complement those, this Spec detail 3 additional templates to support the following flows:
+
+* NFT Sale. An asset owner tokenized an asset and sell one or many of the NFTs to a different user.
+* NFT Access. A NFT holder can get access to exclusive contents for holding that NFT
+* DID Sale. An asset owner wants to sell it totally.
+
+#### Template for selling a tokenized asset via NFTs
+
+The NFT Sales template supports an scenario where an Asset owner wants to tokenized and asset and sell pieces of it via NFTs. Owners buying a new NFT can sell them later to others in a secondary market.
+
+Anyone (consumer/provider/publisher) can use this template in order to setup an agreement allowing a NFT owner to transfer the asset ownership after some payment.
+
+The template is a composite of 3 basic conditions:
+- Lock Payment Condition
+- Transfer NFT Condition
+- Escrow Reward Condition
+
+This scenario takes into account royalties for original creators in the secondary market.
+Once the agreement is created, the consumer after payment can request the transfer of the NFT from the current owner for a specific DID.
+
+The DID Sales template is provided by the `NFTSalesTemplate` Smart Contract.
+
+
+#### Access to contents holding a NFT
+
+The NFT Access template is use case specific template that allows a NFT owner to get access to exclusive contents provided by the original asset creator associated to the NFT.
+Anyone (consumer/provider/publisher) can use this template in order to setup an agreement allowing NFT holders to get access to Nevermined services. The template is a composite of 2 basic conditions:
+- NFT Holding Condition
+- Access Condition
+
+Once the agreement is created, the consumer can demonstrate is holding a NFT for a specific DID. If that's the case the Access condition can be fulfilled by the asset owner or provider and all the agreement is fulfilled.
+This can be used in scenarios where a data or services owner, can allow  users to get access to exclusive services only when they demonstrate the are holding a specific number of NFTs of a DID.
+
+The DID Sales template is provided by the `NFTAccessTemplate` Smart Contract.
+
+
+#### Template for selling an Asset
```

Comment:
It is not clear to me the difference between selling an asset and selling a tokenized asset via NFTs. 
Does this mean selling the asset and all nfts under that DID?

Reply:
Selling the asset means to transfer the ownership of the DID to someone else. Selling a NFT means you only transfer a tokenized part of an Asset but without any implication in the Asset/DID itself. 
After transfering the ownership of a Asset/DID the new owner could mint and burn for example.